src/WinningVoteRegistrar.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+
+namespace PivotLibre\Tideman;
+
+class WinningVoteRegistrar extends PairRegistrar
+{
+
+    /**
+     *
+     * This class...
+     * ...increments the votes of the A->B Pair in the Pair registry if A is preferred over B
+     * ...increments the votes of the B->A Pair in the Pair registry if B is preferred over A
+     * ...increments the votes of both the A->B and B->A Pairs in the Pair registry if A and B are tied
+     * ...increments the indifference of both the A->B and B->A Pairs in the Pair registry if A and B are tied
+     *
```

Comment:
On the continuum of crystal clear to clear as mud, where does this documentation fall? How could it be improved?